<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">ul.lst-kix_27euqc7fls0u-8{list-style-type:none}ul.lst-kix_27euqc7fls0u-7{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-2{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-1{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-0{list-style-type:none}ul.lst-kix_27euqc7fls0u-0{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-6{list-style-type:none}.lst-kix_dphqlby1voqt-6>li{counter-increment:lst-ctn-kix_dphqlby1voqt-6}ol.lst-kix_9hhb75bsnuyf-5{list-style-type:none}ol.lst-kix_dphqlby1voqt-5.start{counter-reset:lst-ctn-kix_dphqlby1voqt-5 0}ul.lst-kix_27euqc7fls0u-2{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-4{list-style-type:none}ul.lst-kix_27euqc7fls0u-1{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-3{list-style-type:none}ul.lst-kix_27euqc7fls0u-4{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-0.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-0 0}ul.lst-kix_27euqc7fls0u-3{list-style-type:none}ul.lst-kix_27euqc7fls0u-6{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-8{list-style-type:none}ul.lst-kix_27euqc7fls0u-5{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-7{list-style-type:none}ul.lst-kix_dtmbe95eiaqv-0{list-style-type:none}ul.lst-kix_dtmbe95eiaqv-1{list-style-type:none}ul.lst-kix_dtmbe95eiaqv-2{list-style-type:none}ol.lst-kix_9hhb75bsnuyf-3.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-3 0}.lst-kix_9hhb75bsnuyf-8>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-8}.lst-kix_27euqc7fls0u-8>li:before{content:"\0025a0   "}ul.lst-kix_dtmbe95eiaqv-7{list-style-type:none}ul.lst-kix_dtmbe95eiaqv-8{list-style-type:none}ul.lst-kix_dtmbe95eiaqv-3{list-style-type:none}ul.lst-kix_dtmbe95eiaqv-4{list-style-type:none}ol.lst-kix_dphqlby1voqt-8.start{counter-reset:lst-ctn-kix_dphqlby1voqt-8 0}ul.lst-kix_dtmbe95eiaqv-5{list-style-type:none}ul.lst-kix_dtmbe95eiaqv-6{list-style-type:none}.lst-kix_cqbtfbwcdrh5-1>li:before{content:"\0025cb   "}.lst-kix_cqbtfbwcdrh5-0>li:before{content:"\0025cf   "}.lst-kix_cqbtfbwcdrh5-4>li:before{content:"\0025cb   "}ol.lst-kix_dphqlby1voqt-2.start{counter-reset:lst-ctn-kix_dphqlby1voqt-2 0}.lst-kix_cqbtfbwcdrh5-2>li:before{content:"\0025a0   "}.lst-kix_cqbtfbwcdrh5-3>li:before{content:"\0025cf   "}.lst-kix_dphqlby1voqt-5>li{counter-increment:lst-ctn-kix_dphqlby1voqt-5}.lst-kix_dphqlby1voqt-8>li{counter-increment:lst-ctn-kix_dphqlby1voqt-8}.lst-kix_9hhb75bsnuyf-7>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-7}.lst-kix_9hhb75bsnuyf-4>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-4,lower-latin) ". "}.lst-kix_9hhb75bsnuyf-3>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-3,decimal) ". "}ol.lst-kix_dphqlby1voqt-3.start{counter-reset:lst-ctn-kix_dphqlby1voqt-3 0}.lst-kix_9hhb75bsnuyf-2>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-2,lower-roman) ". "}.lst-kix_9hhb75bsnuyf-1>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-1,lower-latin) ". "}ol.lst-kix_9hhb75bsnuyf-5.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-5 0}.lst-kix_9hhb75bsnuyf-0>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-0,decimal) ". "}.lst-kix_9hhb75bsnuyf-5>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-5}.lst-kix_27euqc7fls0u-4>li:before{content:"\0025cb   "}.lst-kix_dtmbe95eiaqv-3>li:before{content:"\0025cf   "}.lst-kix_dtmbe95eiaqv-4>li:before{content:"\0025cb   "}ol.lst-kix_dphqlby1voqt-4.start{counter-reset:lst-ctn-kix_dphqlby1voqt-4 0}.lst-kix_dphqlby1voqt-7>li{counter-increment:lst-ctn-kix_dphqlby1voqt-7}.lst-kix_27euqc7fls0u-2>li:before{content:"\0025a0   "}.lst-kix_27euqc7fls0u-6>li:before{content:"\0025cf   "}.lst-kix_dphqlby1voqt-4>li{counter-increment:lst-ctn-kix_dphqlby1voqt-4}ol.lst-kix_9hhb75bsnuyf-6.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-6 0}.lst-kix_dtmbe95eiaqv-1>li:before{content:"\0025cb   "}.lst-kix_dtmbe95eiaqv-5>li:before{content:"\0025a0   "}.lst-kix_27euqc7fls0u-3>li:before{content:"\0025cf   "}.lst-kix_27euqc7fls0u-7>li:before{content:"\0025cb   "}.lst-kix_dphqlby1voqt-1>li{counter-increment:lst-ctn-kix_dphqlby1voqt-1}.lst-kix_dtmbe95eiaqv-2>li:before{content:"\0025a0   "}.lst-kix_27euqc7fls0u-5>li:before{content:"\0025a0   "}.lst-kix_27euqc7fls0u-0>li:before{content:"\0025cf   "}.lst-kix_dtmbe95eiaqv-0>li:before{content:"\0025cf   "}.lst-kix_27euqc7fls0u-1>li:before{content:"\0025cb   "}ol.lst-kix_9hhb75bsnuyf-7.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-7 0}.lst-kix_9hhb75bsnuyf-3>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-3}.lst-kix_dphqlby1voqt-0>li{counter-increment:lst-ctn-kix_dphqlby1voqt-0}.lst-kix_dphqlby1voqt-6>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-6,decimal) ". "}.lst-kix_dphqlby1voqt-7>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-7,lower-latin) ". "}.lst-kix_dphqlby1voqt-5>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-5,lower-roman) ". "}.lst-kix_9hhb75bsnuyf-2>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-2}.lst-kix_dtmbe95eiaqv-7>li:before{content:"\0025cb   "}.lst-kix_dtmbe95eiaqv-8>li:before{content:"\0025a0   "}.lst-kix_dphqlby1voqt-8>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-8,lower-roman) ". "}.lst-kix_dtmbe95eiaqv-6>li:before{content:"\0025cf   "}ol.lst-kix_9hhb75bsnuyf-1.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-1 0}.lst-kix_dphqlby1voqt-1>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-1,lower-latin) ". "}.lst-kix_dphqlby1voqt-2>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-2,lower-roman) ". "}.lst-kix_dphqlby1voqt-3>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-3,decimal) ". "}ol.lst-kix_dphqlby1voqt-6.start{counter-reset:lst-ctn-kix_dphqlby1voqt-6 0}.lst-kix_dphqlby1voqt-4>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-4,lower-latin) ". "}.lst-kix_dphqlby1voqt-2>li{counter-increment:lst-ctn-kix_dphqlby1voqt-2}.lst-kix_9hhb75bsnuyf-1>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-1}ol.lst-kix_9hhb75bsnuyf-4.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-4 0}.lst-kix_9hhb75bsnuyf-4>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-4}.lst-kix_dphqlby1voqt-0>li:before{content:"" counter(lst-ctn-kix_dphqlby1voqt-0,decimal) ". "}.lst-kix_9hhb75bsnuyf-5>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-5,lower-roman) ". "}ul.lst-kix_cqbtfbwcdrh5-0{list-style-type:none}.lst-kix_9hhb75bsnuyf-6>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-6}ul.lst-kix_cqbtfbwcdrh5-1{list-style-type:none}.lst-kix_9hhb75bsnuyf-0>li{counter-increment:lst-ctn-kix_9hhb75bsnuyf-0}.lst-kix_9hhb75bsnuyf-7>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-7,lower-latin) ". "}ul.lst-kix_cqbtfbwcdrh5-4{list-style-type:none}ul.lst-kix_cqbtfbwcdrh5-5{list-style-type:none}.lst-kix_9hhb75bsnuyf-6>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-6,decimal) ". "}ul.lst-kix_cqbtfbwcdrh5-2{list-style-type:none}ul.lst-kix_cqbtfbwcdrh5-3{list-style-type:none}ul.lst-kix_cqbtfbwcdrh5-8{list-style-type:none}.lst-kix_dphqlby1voqt-3>li{counter-increment:lst-ctn-kix_dphqlby1voqt-3}ul.lst-kix_cqbtfbwcdrh5-6{list-style-type:none}ul.lst-kix_cqbtfbwcdrh5-7{list-style-type:none}.lst-kix_9hhb75bsnuyf-8>li:before{content:"" counter(lst-ctn-kix_9hhb75bsnuyf-8,lower-roman) ". "}ol.lst-kix_dphqlby1voqt-5{list-style-type:none}ol.lst-kix_dphqlby1voqt-4{list-style-type:none}.lst-kix_cqbtfbwcdrh5-8>li:before{content:"\0025a0   "}ol.lst-kix_dphqlby1voqt-7{list-style-type:none}ol.lst-kix_dphqlby1voqt-6{list-style-type:none}ol.lst-kix_dphqlby1voqt-1{list-style-type:none}ol.lst-kix_dphqlby1voqt-0{list-style-type:none}ol.lst-kix_dphqlby1voqt-0.start{counter-reset:lst-ctn-kix_dphqlby1voqt-0 0}ol.lst-kix_dphqlby1voqt-3{list-style-type:none}ol.lst-kix_dphqlby1voqt-2{list-style-type:none}.lst-kix_cqbtfbwcdrh5-5>li:before{content:"\0025a0   "}.lst-kix_cqbtfbwcdrh5-6>li:before{content:"\0025cf   "}.lst-kix_cqbtfbwcdrh5-7>li:before{content:"\0025cb   "}ol.lst-kix_9hhb75bsnuyf-8.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-8 0}ol.lst-kix_dphqlby1voqt-8{list-style-type:none}ol.lst-kix_dphqlby1voqt-1.start{counter-reset:lst-ctn-kix_dphqlby1voqt-1 0}ol.lst-kix_dphqlby1voqt-7.start{counter-reset:lst-ctn-kix_dphqlby1voqt-7 0}ol.lst-kix_9hhb75bsnuyf-2.start{counter-reset:lst-ctn-kix_9hhb75bsnuyf-2 0}ol{margin:0;padding:0}table td,table th{padding:0}.c18{padding-top:18pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:justify}.c1{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:20pt;font-family:"Arial";font-style:normal}.c4{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:justify;height:11pt}.c0{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c14{padding-top:0pt;padding-bottom:3pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:justify}.c11{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c3{padding-top:20pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:justify}.c13{padding-top:12pt;text-indent:36pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c5{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c15{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c12{font-weight:700;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c6{font-weight:400;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c9{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline}.c2{background-color:#ffffff;color:#222222;font-size:10pt}.c17{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c8{color:inherit;text-decoration:inherit}.c19{color:#000000;font-size:16pt}.c10{background-color:#ffffff;font-size:10pt}.c21{color:#000000;font-size:26pt}.c7{font-style:italic}.c16{font-size:10pt}.c20{text-indent:36pt}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c17 doc-content"><p class="c14 title" id="h.b2cp0ns8tx1o"><span class="c6 c21">Simulating metagenomic data for a novel pathogen outbreak detection</span></p><p class="c5"><span class="c0">Marie Kr&aacute;tk&aacute;</span></p><h1 class="c3" id="h.ua6onbg1ei21"><span>Introduction</span></h1><p class="c5"><span class="c0">The detection of outbreaks caused by a novel pathogen usually depends on healthcare professionals noticing and reporting unusual cases. However, this approach can be inefficient and slow, particularly for diseases with a high proportion of asymptomatic infections (e.g. poliovirus) or delayed symptom onset (e.g. HIV/AIDS). Such &ldquo;stealth outbreaks&rdquo; can spread undetected for a longer period, delaying public health response and complicating containment efforts.</span></p><p class="c4"><span class="c0"></span></p><p class="c5"><span>To address the challenges posed by stealth outbreaks, it is crucial to develop early warning systems that do not depend on symptomatic patient reports. One promising approach is to leverage environmental surveillance. Even asymptomatic carriers often shed pathogen particles, which can be detected in samples from strategically selected locations, such as metropolitan areas or high-volume transit hubs. Sample types may include wastewater, air, or pooled specimens like nasal swabs and blood samples. Sequencing these samples produces metagenomic profiles detailing the composition of organism communities, enabling the identification of outbreaks by detecting a pathogen&#39;s increasing abundance over time, even without prior knowledge of the pathogen.</span></p><p class="c4"><span class="c0"></span></p><p class="c5"><span class="c0">Developing reliable workflows to detect such signals requires datasets where outbreak scenarios are explicitly modeled. However, obtaining such datasets from natural conditions presents significant challenges: it demands extensive sampling, and the exact dynamics of an outbreak&mdash;such as the proportion of infected individuals contributing to the samples&mdash;are often unknown. Moreover, to ensure the robustness of detection workflows, it is necessary to test them under diverse conditions, including varying pathogen characteristics and transmission parameters.</span></p><p class="c4"><span class="c0"></span></p><p class="c5"><span class="c0">Simulated datasets offer a practical alternative. By controlling the parameters of the outbreak and the dataset, it should be possible to systematically explore different scenarios and validate data analysis workflows. This work outlines and demonstrates a methodology for creating a simulated dataset of a viral outbreak. First, we selected and processed a time series of metagenomic wastewater data from a public database to use as a realistic background. Next, we chose a pathogen genome from a public repository, and generated computer-simulated sequencing reads of the pathogen. Finally, we modeled pathogen abundance over time based on an exponential growth curve, and combined the background and pathogen reads in the calculated ratios.</span></p><p class="c4"><span class="c0"></span></p><p class="c5"><span>Beyond practically implementing these steps, we prioritized outlining the general guidelines and key considerations of simulating such metagenomic datasets. This facilitates further development of the workflow and adaptation of the inputs and the</span><span class="c0">&nbsp;methodology for simulation of various outbreak scenarios.</span></p><h1 class="c3" id="h.wzsd0b1mt75"><span class="c1">Methods and Results</span></h1><p class="c5"><span class="c0">Fig. 1: Overview of the workflow for generating simulated outbreak metagenomic dataset</span></p><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 480.50px; height: 529.01px;"><img alt="" src="images/image1.png" style="width: 480.50px; height: 529.01px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c18" id="h.olrslgg3l5dt"><span class="c6 c19">Background metagenomic data</span></h2><p class="c5"><span>We downloaded wastewater metagenomic dataset published in Rothman </span><span class="c7">et al.</span><span>&nbsp;2021 (1) mapping RNA virome in Southern California from August 2020 to January 2021. Since this dataset contains samples from multiple water treatment plants and even different methods of sequencing library preparation, we use only data from one of the plants (HTP) and the library that underwent only non-specific PCR amplification (isn&rsquo;t enriched for any particular panel of sequences). We downloaded selected samples</span><span>&nbsp;using the NCBI SRA Toolkit (2) and then assessed the quality of the reads using FastQC and MultiQC tools (3, 4). We found that the number of reads ranged from 2 to 89 M pairs among the 92 samples. As expected, these unprocessed reads contain sequencing </span><span>adapters</span><span>. Non-specific amplification during library preparation also produced significant content of duplicated reads (Fig. 2). Four samples encountered </span><span>unexpected memory or parsing errors</span><span>&nbsp;</span><span>when processed by quality control tools and were excluded from further processing. Code for this step is available in the project repository (5) at </span><span class="c9"><a class="c8" href="https://www.google.com/url?q=https://github.com/437364/outbreak_metagenomic_sim/blob/main/workflow/rules/download_background.smk&amp;sa=D&amp;source=editors&amp;ust=1736730193713226&amp;usg=AOvVaw1CcO5LstGdiRmkNABxaGjS">rules/download_background.smk</a></span><span>. For the preparation of the simulated dataset, we further excluded samples with fewer than 5 million reads and then selected 20 consecutive samples with less than 7-day intervals to ensure that all stages of the exponential growth curve within the given timeframe are sufficiently covered.</span></p><p class="c4"><span class="c0"></span></p><p class="c5"><span class="c0">Fig. 2: Boxplot of sample read counts and proportion of duplicated reads in samples</span></p><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 483.50px; height: 330.86px;"><img alt="" src="images/image3.png" style="width: 483.50px; height: 330.86px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h2 class="c18" id="h.hu4chk0jitk"><span class="c6 c19">Pathogen data</span></h2><p class="c5"><span>Next, we downloaded a reference genome of a suitable pathogen. For this demonstration, we chose Nipah virus, since this RNA virus causing lethal encephalitis is not expected to be present in the background data. We generated </span><span>100&rsquo;000</span><span>&nbsp;computer-simulated Illumina reads using InSilicoSeq (6). This tool does not allow modification of the read size, defaulting to 150 base pairs, which corresponds to the read length of a portion of the chosen background datasets. These computer-simulated reads lack adapter sequences. For this demonstration, we did not attempt to include adapters, but this might be a useful feature to be added to the workflow later (7) to better simulate the background dataset sequencing. Code for this step is available in the project repository (5) at </span><span class="c9"><a class="c8" href="https://www.google.com/url?q=https://github.com/437364/outbreak_metagenomic_sim/blob/main/workflow/scripts/simulate_pathogen_reads.sh&amp;sa=D&amp;source=editors&amp;ust=1736730193714180&amp;usg=AOvVaw3Uozzcti5y2U7nAjN11qQM">scripts/simulate_pathogen_reads.sh</a></span></p><h2 class="c18" id="h.yaqzcm525xrb"><span class="c6 c19">Growth calculation and subsampling</span></h2><p class="c5"><span>Then, we calculated the expected abundance of pathogen reads in the metagenomic data time series. The amount of viral reads is low in shotgun metagenomic data from wastewater samples. While methodologies to increase the proportion by PCR amplification exist, they could make the detection method less sensitive to very diverged (e.g. engineered) viruses. Previous studies have shown that the estimated hypothetical abundance of viruses in wastewater samples if 100% of the contributing population was infected at the same time is about </span><span>0.0006%</span><span class="c0">&nbsp;(6 pathogen reads per 1&rsquo;000&rsquo;000 total) for SARS-CoV-2 and 0.1% (1 pathogen read per 1&rsquo;000 total) for a norovirus (8). For this demonstration, we chose the value of this parameter (also referred to as conversion factor B) 0.01%, which is inside the previously reported range. &nbsp; &nbsp; </span></p><p class="c5"><span class="c0">Further, we selected 10 million reads as the expected sample size for the simulated dataset, since this is a sufficient size for detecting some pathogen reads in wastewater metagenomic samples, but still reasonable in terms of computational resource requirements and processing speed. Additional parameters such as the reproduction number and generation time were manually selected for achieving ~5% infection prevalence in the simulated dataset, but can be easily modified to simulate scenarios with variable outbreak dynamics. This simplified model does not take into account parameters such as the reproduction number changing over time as the number of susceptible hosts decreases or the duration it takes an infected person to recover, making this model less realistic but still sufficient for modeling low prevalence stages (&lt;0.5% infected), where researchers are the most likely to use early detection workflows. </span></p><p class="c4"><span class="c0"></span></p><p class="c5"><span>Expected pathogen abundance was then used to calculate the number of background and pathogen reads in the sample of a given size using selection from a Poisson distribution (Fig. 3). The number of pathogen reads in the samples ranges from 0 to 55 reads. Finally, we subsampled the background and pathogen reads based on this model using </span><span>seqtk</span><span>&nbsp;tool (9). The dataset created in this demonstration is available at Zenodo </span><span>(10</span><span>). Code for this step is available in the project repository (5) at </span><span class="c9"><a class="c8" href="https://www.google.com/url?q=https://github.com/437364/outbreak_metagenomic_sim/blob/main/workflow/notebooks/calculate_outbreak_growth.ipynb&amp;sa=D&amp;source=editors&amp;ust=1736730193715207&amp;usg=AOvVaw3D1GNoAWH6dmEELuUIj6kW">notebooks/calculate_outbreak_growth.ipynb</a></span><span>&nbsp;and </span><span class="c9"><a class="c8" href="https://www.google.com/url?q=https://github.com/437364/outbreak_metagenomic_sim/blob/main/workflow/scripts/subsample_and_combine_reads.sh&amp;sa=D&amp;source=editors&amp;ust=1736730193715420&amp;usg=AOvVaw1PLe2-23pmQk_2LecTqjg9">scripts/subsample_and_combine_reads.sh</a></span></p><p class="c4"><span class="c0"></span></p><p class="c5"><span class="c0">Fig. 3: Simulated pathogen abundance in the metagenomic time series dataset </span></p><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 536.50px; height: 367.12px;"><img alt="" src="images/image2.png" style="width: 536.50px; height: 367.12px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><h1 class="c3" id="h.kfk08q3c1cxk"><span>Discussion</span></h1><p class="c15"><span class="c0">In this project, we demonstrated a methodology for creating simulated metagenomic datasets with exponentially increasing pathogen abundance. These datasets can be used to develop and test workflows to detect novel pathogen outbreaks. Simulated metagenomic datasets also allow convenient performance comparison of different detection workflow variants such as reference-based versus reference-free approaches (11). This facilitates development of a workflow that can reliably detect outbreaks of a wide range of pathogens even in datasets with limited sample size and avoid false positives originating from animal pathogens or other natural sequences.</span></p><p class="c15"><span>To enhance our workflow and enable simulation of a wider range of outbreak scenarios, several improvements and extensions can be considered (described in more detail in the project repository </span><span class="c9"><a class="c8" href="https://www.google.com/url?q=https://github.com/437364/outbreak_metagenomic_sim/issues&amp;sa=D&amp;source=editors&amp;ust=1736730193716226&amp;usg=AOvVaw1Wqn2MOf8Nq56fO17Rntf5">&ldquo;Issues&rdquo; section</a></span><span class="c0">):</span></p><p class="c13"><span class="c0">1. &nbsp;Unprocessed background metagenomic datasets contain technical artifacts such as read duplication and presence of adapters, which would be typically addressed during the preprocessing stage of detection workflows. Adding simulated pathogen sequences (which do not contain these artifacts) before the preprocessing might result in a skewed sample composition profile. Adding artifacts to the pathogen reads could address this issue. </span></p><p class="c5 c20"><span class="c0">2. In the computer-simulated sequencing reads, parameters such as the read length, fragment length, and sequencing error rate should correspond to the background data. We could develop methods to infer these parameters from background data and apply them to the computer-simulated sequencing.</span></p><p class="c4"><span class="c0"></span></p><p class="c5 c20"><span class="c0">3. Due to low abundance of viral reads in shotgun wastewater samples, PCR amplification is often used to aid detection. Simulation of this kind of datasets should be considered. </span></p><p class="c4"><span class="c0"></span></p><p class="c5 c20"><span class="c0">4. Current workflow uses a simplified model of a pathogen outbreak. Including variables such as gradual decrease of viral particle shedding by recovering individuals and fluctuations of the reproduction number would make the simulations more realistic. Additionally, introducing random noise to the model could also further increase real-world faithfulness of the simulated datasets.</span></p><p class="c4"><span class="c0"></span></p><p class="c5"><span class="c0">Developing functional growth detection workflows is a complex, multifaceted task. I hope researchers tackling this challenge will find my work on dataset simulation valuable as a practical starting point. By focusing on workflow documentation, customization, and potential enhancements, I aimed to provide a solid foundation for generating simulated metagenomic datasets, ultimately supporting advances in outbreak detection and public health response.</span></p><h1 class="c3" id="h.2ixfokcamuec"><span class="c1">Acknowledgement</span></h1><p class="c5"><span>I&rsquo;d like to thank Mike McLaren, Evan Fields, and Dan Rice from the </span><span class="c9"><a class="c8" href="https://www.google.com/url?q=https://naobservatory.org/&amp;sa=D&amp;source=editors&amp;ust=1736730193717302&amp;usg=AOvVaw1MRWkneQAHJGJPx4HZMsWY">Nucleic Acid Observatory</a></span><span class="c0">&nbsp;project for their valuable and abundant guidance and feedback.</span></p><h1 class="c3" id="h.kikl1cao19zq"><span class="c1">References</span></h1><p class="c5"><span>1. </span><span class="c2">Rothman, J. A., Loveless, T. B., Kapcia III, J., Adams, E. D., Steele, J. A., Zimmer-Faust, A. G., ... &amp; Whiteson, K. L. (2021). </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1128/aem.01448-21&amp;sa=D&amp;source=editors&amp;ust=1736730193717673&amp;usg=AOvVaw0DGheUUX8oGb3yreMLLCmB">RNA viromics of Southern California wastewater and detection of SARS-CoV-2 single-nucleotide variants. </a></span><span class="c9 c10 c7"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1128/aem.01448-21&amp;sa=D&amp;source=editors&amp;ust=1736730193717791&amp;usg=AOvVaw3PKLGcRA8gk9Xzht1Asl0h">Applied and environmental microbiology</a></span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1128/aem.01448-21&amp;sa=D&amp;source=editors&amp;ust=1736730193717886&amp;usg=AOvVaw3hadx5oz5k5FZbv4208dE7">,</a></span><span class="c2">&nbsp;</span><span class="c2 c7">87</span><span class="c6 c2">(23), e01448-21.</span></p><p class="c5"><span class="c2">2. </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://github.com/ncbi/sra-tools&amp;sa=D&amp;source=editors&amp;ust=1736730193718151&amp;usg=AOvVaw1VMWzK8OxwJ0nXFB4qWgSR">https://github.com/ncbi/sra-tools</a></span></p><p class="c5"><span class="c2">3. Andrews, S. (2010, April). </span><span class="c9 c10 c7"><a class="c8" href="https://www.google.com/url?q=https://www.bioinformatics.babraham.ac.uk/projects/fastqc/&amp;sa=D&amp;source=editors&amp;ust=1736730193718393&amp;usg=AOvVaw22KlchJ_vfdn8L_erbA8gA">FastQC: a quality control tool for high throughput sequence data</a></span><span class="c6 c2">.</span></p><p class="c5"><span class="c2">4. Ewels, P., Magnusson, M., Lundin, S., &amp; K&auml;ller, M. (2016). </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1093/bioinformatics/btw354&amp;sa=D&amp;source=editors&amp;ust=1736730193718620&amp;usg=AOvVaw1M-IQ0jps3rVHS3ou31VL5">MultiQC: summarize analysis results for multiple tools and samples in a single report</a></span><span class="c2">. </span><span class="c2 c7">Bioinformatics</span><span class="c2">, </span><span class="c2 c7">32</span><span class="c6 c2">(19), 3047-3048.</span></p><p class="c5"><span class="c2">5. </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://github.com/437364/outbreak_metagenomic_sim&amp;sa=D&amp;source=editors&amp;ust=1736730193718976&amp;usg=AOvVaw3vuVyKFgtUzXCRgAogPV5g">https://github.com/437364/outbreak_metagenomic_sim</a></span></p><p class="c5"><span class="c2">6. Gourl&eacute;, H., Karlsson-Lindsj&ouml;, O., Hayer, J., &amp; Bongcam-Rudloff, E. (2019). </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1093/bioinformatics/bty630&amp;sa=D&amp;source=editors&amp;ust=1736730193719215&amp;usg=AOvVaw3JTIISxIjLJptbQJPubtfQ">Simulating Illumina metagenomic data with InSilicoSeq</a></span><span class="c2">. </span><span class="c2 c7">Bioinformatics</span><span class="c2">, </span><span class="c2 c7">35</span><span class="c6 c2">(3), 521-522.</span></p><p class="c5"><span class="c2">7. </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://github.com/437364/outbreak_metagenomic_sim/issues/1&amp;sa=D&amp;source=editors&amp;ust=1736730193719576&amp;usg=AOvVaw0CCau8nH0UInBE79UXpB8x">https://github.com/437364/outbreak_metagenomic_sim/issues/1</a></span></p><p class="c5"><span class="c2">8. Grimm, S. L., Kaufman, J. T., Rice, D. P., Whittaker, C., Bradshaw, W. J., &amp; McLaren, M. R. (2023). </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.1101/2023.12.22.23300450&amp;sa=D&amp;source=editors&amp;ust=1736730193719823&amp;usg=AOvVaw14XC_o0VjWlxvHf478yPLd">Inferring the sensitivity of wastewater metagenomic sequencing for pathogen early detection.</a></span><span class="c2">&nbsp;</span><span class="c2 c7">medRxiv</span><span class="c6 c2">, 2023-12.</span></p><p class="c5"><span class="c2">9. Li, H. (2023). seqtk </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://github.com/lh3/seqtk&amp;sa=D&amp;source=editors&amp;ust=1736730193720091&amp;usg=AOvVaw3p2xukL3nSBe7VCfL71zDl">Toolkit for processing sequences in FASTA/Q formats</a></span><span class="c2">. </span><span class="c2 c7">GitHub</span><span class="c2">, </span><span class="c2 c7">767</span><span class="c6 c2">, 69.</span></p><p class="c5"><span class="c2">10. </span><span class="c9 c16"><a class="c8" href="https://www.google.com/url?q=https://zenodo.org/records/14635401&amp;sa=D&amp;source=editors&amp;ust=1736730193720423&amp;usg=AOvVaw1g9snFJQyxuB0jJN5sYaz3">https://zenodo.org/records/14635401</a></span></p><p class="c5"><span class="c2">11. Consortium, T. N. A. O. (2021). </span><span class="c9 c10"><a class="c8" href="https://www.google.com/url?q=https://doi.org/10.48550/arXiv.2108.02678&amp;sa=D&amp;source=editors&amp;ust=1736730193720652&amp;usg=AOvVaw2jA2gepVX9_KF15m4P1fYg">A global nucleic acid observatory for biodefense and planetary health</a></span><span class="c2">. </span><span class="c2 c7">arXiv preprint arXiv:2108.02678</span><span class="c2">.</span></p><p class="c4"><span class="c0"></span></p><p class="c4"><span class="c0"></span></p></body></html>